examples/realworld/ext/article/queries.js

```diff
@@ -84,6 +84,9 @@ export const getArticle = async ({ slug }, context) => {
     where: { slug },
     include: articleInclude
   })
+  if (!article) { 
+      throw new HttpError(404)
```

Comment:
Nice! Do notice though that indentation everywhere is 2 while here it is 4 :P.

Reply:
I'll blame this on the editor. I'll fix it.